From f5859057035f60966fade824691928bfe2c472e0 Mon Sep 17 00:00:00 2001
From: Jon Turney <jon.turney@dronecode.org.uk>
Date: Thu, 13 Feb 2014 01:06:51 +0000
Subject: [PATCH 09/23] Fix common.gypi to allow gyp to generate Makefiles on
 Windows

The MSVS_VERSION variable does not exist unless we are using the MSVS generator

Signed-off-by: Jon Turney <jon.turney@dronecode.org.uk>
---
 src/build/common.gypi | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/build/common.gypi b/src/build/common.gypi
index b9466a3..9befd15 100644
--- a/src/build/common.gypi
+++ b/src/build/common.gypi
@@ -206,9 +206,9 @@
       # Whether to use multiple cores to compile with visual studio. This is
       # optional because it sometimes causes corruption on VS 2005.
       # It is on by default on VS 2008 and off on VS 2005.
-      ['OS=="win"', {
+      ['"<(GENERATOR)" == "msvs"', {
         'conditions': [
-          ['MSVS_VERSION=="2005"', {
+          [ 'MSVS_VERSION=="2005"', {
             'msvs_multi_core_compile%': 0,
           },{
             'msvs_multi_core_compile%': 1,
@@ -225,6 +225,10 @@
           # Native Client loader for 64-bit Windows.
           'NACL_WIN64',
         ],
+      },
+      {
+            # XXX: because value is used below...
+            'msvs_multi_core_compile%': 0,
       }],
     ],
 
-- 
2.7.0

